src/main/java/uk/gov/hmcts/reform/sendletter/tasks/SerialTaskRunner.java

```diff
@@ -55,6 +56,11 @@ public void tryRun(int id, Runnable runnable) throws SQLException {
                     }
                 }
             }
+        } catch (SQLException exception) {
+            log.error(
+                String.format("SQL error occurred during task %s run", Task.getTaskFromLockId(id)),
+                new TaskRunnerException(exception)
```

Comment:
why not `exception`?

Reply:
Because hmcts loggers require alert levels to be implemented